app/src/main/java/ru/yandex/practicum/contacts/presentation/filter/model/FilterContactTypeUi.java

```diff
@@ -19,6 +21,10 @@ public FilterContactType getContactType() {
     public boolean isSelected() {
         return selected;
     }
+    @Override
+    public boolean theSameAs (FilterContactTypeUi other) {
```

Comment:
Перед скобкой здесь лишний пробел:
```suggestion
    public boolean theSameAs(FilterContactTypeUi other) {
```
В IDE можно сделать авто форматирование, нажав ПКМ по нужному классу и выбрав `Reformat code`, или нажать сочетание клавиш из соответствующего пункта в меню.